man/man7/vdevprops.7

```diff
@@ -121,6 +121,9 @@ dataset.
 A text comment up to 8192 characters long
 .It Sy bootsize
 The amount of space to reserve for the EFI system partition
+.It Sy failfast
+If this device should propage BIO errors back to ZFS, used to disable
+failfast.
```

Comment:
How about this instead since any unrecoverable errors will always be propagated back to ZFS.

"If device/host/transport errors should be retried at the block layer or instead fail fast."

Reply:
Fixed.